src/prefect/agent/local/agent.py

```diff
@@ -67,16 +76,21 @@ def deploy_flows(self, flow_runs: list) -> None:
             if not self.no_pull and storage.registry_url:
                 self.logger.debug("Pulling image {}...".format(storage.name))
                 try:
-                    self.docker_client.pull(storage.name)
+                    pull_output = self.docker_client.pull(storage.name)
```

Comment:
I vote that we stream this output just like we do in the Docker storage class, and log it as it arrives.  This would help us debug that annoying Windows machine situation where the pull takes forever.